Triton_Inference_Server_Python_API/examples/kafka-io/start-server.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+export KAFKA_CONSUMER_MAX_WORKER_THREADS=1
+export CONSUMER_CONFIGS='{"bootstrap.servers": "localhost:9092", "security.protocol": "PLAINTEXT", "group.id": "triton-server-kafka-consumer"}'
+export PRODUCER_CONFIGS='{"bootstrap.servers": "localhost:9092", "security.protocol": "PLAINTEXT"}'
+export CONSUMER_TOPICS='inference-input'
+export PRODUCER_TOPIC='inference-output'
+export MODEL_INPUT_NAME='TEXT'
+export MODEL_NAME='tokenizer'
+export MODEL_REPOSITORY='./models'
+
+nohup serve run tritonserver_deployment:entrypoint &
```

Comment:
```suggestion
nohup serve run tritonserver_deployment:entrypoint &
tailf -f nohup.out
```
As discussed, let's have users monitor the log in order to tell when the server is live. On my side it looks like they can begin producing when they see something like "024-07-18 21:55:38,254 INFO api.py:609 -- Deployed app 'default' successfully."

Reply:
Updated